packages/plugins/src/SidebarPlugin.ts

```diff
@@ -172,14 +152,18 @@ const SidebarPlugin: PluginType<SidebarPluginPage, SidebarPluginOptions, Sidebar
         deep: sidebarRootLevel
       });
 
-      //Map into Sidebar Groups and sort children according to priority, priority = 1 has the highest priority
+      //Map into Sidebar Groups and sort children according to priority
       function sortSidebarGroups(sidebarData) {
         const sortedGroupedPages = sidebarData.map(page => {
-          if (page.childNodes) {
-            const sortedPages = page.childNodes.sort((a, b) => b.priority - a.priority);
+          if (page.childNodes.length > 1) {
+            const sortedPages = page.childNodes.sort(
+              (pageA, pageB) =>
+                (pageB.priority ? pageB.priority : -1) - (pageA.priority ? pageA.priority : -1)
+            );
+            sortSidebarGroups(page.childNodes);
             return { ...page, childNodes: sortedPages };
           } else {
-            page;
+            return page;
```

Comment:
good spot... this is why it's a good idea to add tests, we would of seen this issue sooner.
our test coverage is improving, so consider adding tests, as we go.

not to say we need to block this PR as it's missing tests but always nice to see a PR with test coverage.